scripts/config/!msu.nut

```diff
@@ -5,3 +5,24 @@
 	VanillaID = "vanilla",
 	Class = {}
 };
+
+::includeLoad <- function( _prefix, _folder )
+{
+	::include(_prefix + _folder + "/load.nut");
+}
+::includeFile <- function( _prefix, _file )
+{
+	::include(_prefix + _file);
+}
+::includeFiles <- function( _files, _includeLoad = false )
+{
+	foreach (file in _files)
+	{
+		if (_includeLoad || split(file, "/").pop() != "load.nut")
+		{
+			::include(file);
+		}
+	}
+}
+
+::includeFiles(::IO.enumerateFiles("scripts/config/msu"));
```

Comment:
As we have `!msu.nut` in the `scripts/config` folder, perhaps it is best to move the Classes etc. that we moved to that folder for the idea of "loading them before hooks" back to the `/msu` folder and instead use this method to load them early i.e. during config load?